db/migrate/20220220190037_devise_token_auth_create_users.rb

```diff
@@ -53,5 +53,6 @@ def change
     add_index :users, :reset_password_token, unique: true
     add_index :users, :confirmation_token,   unique: true
     add_index :users, :unlock_token,         unique: true
+    add_index :users, :username,             unique: true
```

Comment:
should everyone roll back this migration and run it again?
or are there other solutions?